README.md

```diff
@@ -64,8 +64,8 @@ Have a better idea for how to turn features into predictions?
 - 🚨 Important: Pre-trained adaptors are not allowed! Be original — you can use the few-shots, for example, for fitting or training your adaptor, but don’t rely on pre-trained solutions
 
 ### Dependencies
-- Each method must be able to run in the [provided isolated environment](https://github.com/DIAGNijmegen/unicorn_eval/blob/improve-code-readability/Dockerfile)
-- Additional dependencies can be requested but:
+- Each method must be able to run in the [provided isolated environment](https://github.com/DIAGNijmegen/unicorn_eval/blob/main/Dockerfile)
```

Comment:
You can link to the Dockerfile in the current branch using `[provided isolated environment](Dockerfile)`